content/en/docs/examples/microservices-istio/bookinfo-kubernetes/index.md

```diff
@@ -0,0 +1,210 @@
+    details-v1-6d86fd9949-q8rrf     1/1     Running   0          10s
+    productpage-v1-c9965499-tjdjx   1/1     Running   0          8s
+    ratings-v1-7bf577cb77-pq9kg     1/1     Running   0          9s
+    reviews-v1-77c65dc5c6-kjvxs     1/1     Running   0          9s
+    {{< /text >}}
+
+1.  After the four services achieve the `Running` status, you can scale the deployment. To let each version of each microservice run in three pods, execute the following command:
+
+    {{< text bash >}}
+    $ kubectl scale deployments --all --replicas 3
+    deployment "details-v1" scaled
+    deployment "productpage-v1" scaled
+    deployment "ratings-v1" scaled
+    deployment "reviews-v1" scaled
+    deployment "reviews-v2" scaled
+    deployment "reviews-v3" scaled
+    {{< /text >}}
+
+1.  Check the pods status. Notice that each microservice has three pods:
+
+    {{< text bash >}}
+    $ kubectl get pods
+    NAME                            READY   STATUS    RESTARTS   AGE
+    details-v1-6d86fd9949-fr59p     1/1     Running   0          50s
+    details-v1-6d86fd9949-mksv7     1/1     Running   0          50s
+    details-v1-6d86fd9949-q8rrf     1/1     Running   0          1m
+    productpage-v1-c9965499-hwhcn   1/1     Running   0          50s
+    productpage-v1-c9965499-nccwq   1/1     Running   0          50s
+    productpage-v1-c9965499-tjdjx   1/1     Running   0          1m
+    ratings-v1-7bf577cb77-cbdsg     1/1     Running   0          50s
+    ratings-v1-7bf577cb77-cz6jm     1/1     Running   0          50s
+    ratings-v1-7bf577cb77-pq9kg     1/1     Running   0          1m
+    reviews-v1-77c65dc5c6-5wt8g     1/1     Running   0          49s
+    reviews-v1-77c65dc5c6-kjvxs     1/1     Running   0          1m
+    reviews-v1-77c65dc5c6-r55tl     1/1     Running   0          49s
+    {{< /text >}}
+
+1.  Deploy a testing pod, [sleep]({{< github_tree >}}/samples/sleep), to use it for sending
+    requests to your microservices:
+
+    {{< text bash >}}
+    $ kubectl apply -f {{< github_file >}}/samples/sleep/sleep.yaml
+    {{< /text >}}
+
+1.  To confirm that the Bookinfo application is running, send a request to it with a curl command from your testing pod:
+
+    {{< text bash >}}
+    $ kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') -c sleep -- curl productpage:9080/productpage | grep -o "<title>.*</title>"
+    <title>Simple Bookstore App</title>
+    {{< /text >}}
+
+## Enable external access to the application
+
+Once your application is running, enable clients from outside the cluster to access it. Such clients are known as mesh-external clients. Once you
+configure the steps below successfully, you can access the application from your laptop's browser.
+
+{{< warning >}}
+
+If your cluster runs on GKE, change the `productpage` service type to `LoadBalancer` before you create your Kubernetes ingress, as shown in this example:
+
+    {{< text yaml >}}
```

Comment:
This nested code-block is not generating the correct HTML and that's causing the linter error.
Remove the callout to resolve the issue.

Reply:
You should be able to put nested code blocks in a warning, we do in other places.

I think the problem is the indentation level of the text block. It should be at the same level as the text above it.

Reply:
fixed